src/pages/community.js

```diff
@@ -0,0 +1,676 @@
+  display: flex;
+  flex-direction: row-reverse;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    flex-direction: column-reverse;
+    align-items: center;
+  }
+`
+
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    flex-direction: column-reverse;
+    align-items: center;
+  }
+`
+
+const CentralColumn = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 2rem;
+`
+
+const ImageContainer = styled.div`
+  background: "#F1FFFD";
+  display: flex;
+  height: 100%;
+  width: 100%;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    width: 75%;
+  }
+`
+
+const IntroImage = styled(Img)`
+  width: 100%;
+  background-size: cover;
+  background: no-repeat 50px;
+`
+
+const FeatureImage = styled(Img)`
+  width: 100%;
+`
+
+const Subtitle = styled.div`
+  margin-bottom: 2rem;
+  font-size: 20px;
+  line-height: 140%;
+  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
+    font-size: 16px;
+  }
+`
+
+const H2 = styled.h2`
+  margin-top: 0rem;
+`
+
+const H3 = styled.h3`
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 700;
+  text-align: left;
+  margin-top: 0;
+  margin-bottom: 1rem;
+`
+
+const CenterText = styled.p`
+  text-align: center;
+  max-width: 800px;
+  margin-bottom: 1rem;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    margin: auto 1.5rem;
+    margin-bottom: 1rem;
+  }
+`
+
+const OpenSourceContainer = styled.div`
+  background: ${(props) => props.theme.colors.homeBoxTurquoise};
+  display: flex;
+  align-items: center;
+  flex-direction: row-reverse;
+  padding-left: 2rem;
+  width: 100%;
+  height: 720px;
+  margin-top: -1px;
+  border-top: 1px solid ${(props) => props.theme.colors.text};
+  border-bottom: 1px solid ${(props) => props.theme.colors.text};
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    flex-direction: column-reverse;
+    height: 100%;
+    padding-top: 2rem;
+    padding-left: 0rem;
+    padding-bottom: 2rem;
+  }
+`
+
+const PoapContainer = styled.div`
+  background: ${(props) => props.theme.colors.homeBoxPink};
+  display: flex;
+  align-items: center;
+  flex-direction: row-reverse;
+  padding-left: 2rem;
+  height: 720px;
+  width: 100%;
+  margin-top: -1px;
+  margin-bottom: 0rem;
+  border-top: 1px solid ${(props) => props.theme.colors.text};
+  border-bottom: 1px solid ${(props) => props.theme.colors.text};
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    flex-direction: column-reverse;
+    height: 100%;
+    padding-top: 2rem;
+    padding-left: 0rem;
+    padding-bottom: 2rem;
+  }
+`
+
+const SupportContainer = styled.div`
+  background: ${(props) => props.theme.colors.homeBoxPurple};
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  height: 720px;
+  width: 100%;
+  margin-top: -1px;
```

Comment:
This container seems a little crowded on the top in mobile
![Screen Shot 2021-12-06 at 3 08 44 PM](https://user-images.githubusercontent.com/15589226/144930797-47851b2c-0098-492d-ba86-73905f121f95.png)